Scrum 35 UI create project skeleton

AllisterLopesDev opened the pull request:
Initial Ui structure

allister140 committed:
feat: initialize Expo app with basic structure and components

- Add .gitignore to exclude unnecessary files and directories
- Create README.md with setup instructions and project overview
- Initialize app.json with Expo configuration
- Implement tab navigation layout in _layout.tsx
- Create Home and Explore screens with example content
- Add modal screen for navigation demonstration
- Implement collapsible sections for better content organization
- Create themed components for consistent styling
- Add utility hooks for color scheme and theme color management
- Include necessary assets for icons and splash screens
- Set up ESLint configuration for code quality
- Add TypeScript configuration for type safety
- Implement reset script for project initialization

allister140 committed:
feat: initialize STS project with essential configurations and reset script

- Added package.json with dependencies and scripts for the STS project.
- Implemented reset-project.js script to reset the project structure and manage existing files.
- Created tsconfig.json for TypeScript configuration with strict mode and path aliases.

darronmoraes commented on Frontend/STS/app/(tabs)/account.tsx:
```diff
@@ -0,0 +1,35 @@
+import { Text } from 'react-native';
+
+
+export default function TabTwoScreen() {
```
TabTwoScreen? 
Naming of function sounds vague.

darronmoraes commented on Frontend/STS/app/(tabs)/index.tsx:
```diff
@@ -0,0 +1,20 @@
+import { IconSymbol } from '@/components/ui/icon-symbol.ios';
+
+
+export default function HomeScreen() {
```
shouldn't the file name and function name go along, to keep things aligned.

darronmoraes commented on Frontend/STS/package-lock.json:
should be added to git ignore

darronmoraes replied:
add to .gitignore and remove from tracking too

allister140 committed:
Refactor project structure: update package.json, remove reset script, and delete tsconfig.json

- Updated package.json to change the main entry point and streamline scripts.
- Removed the reset-project.js script as it is no longer needed.
- Deleted tsconfig.json to simplify the project setup.

AllisterLopesDev commented:
@darronmoraes @surojigirish  check for [Refactor project structure:](https://github.com/AllisterLopesDev/smart-transit-service/pull/5/commits/9d8e76723373e83f9928e4ad55c5b434a5351231)

allister140 committed:
removed package-lock.json

darronmoraes reviewed (approved):
Approved

AllisterLopesDev commented:
@darronmoraes @surojigirish you guys get this option to merge?
<img width="915" height="169" alt="Screenshot 2025-10-05 221607" src="https://github.com/user-attachments/assets/ac65b165-5b4d-4cd3-920d-20ac105b7c93" />
